swaywm/wlroots
# xdg-shell stable implementation

Roadmap:
* [x] Implement xdg-shell stable
* ~~share as much code as possible with xdg-shell-v6~~

Test plan:
 * Run some glfw examples, check with WAYLAND_DEBUG that they use the new interface (xdg_wm_base etc) and that things appear to work™
 * Would like to test popups as well somehow, not aware of any way to test right now
 * Check I didn't break xdg-shell-v6 as well

Fixes #557
